inf-clojure.el

```diff
@@ -391,12 +384,20 @@ Fallback to `default-directory.' if not within a project."
           ((file-exists-p "build.boot") "boot")
           (t nil))))
 
+(defun inf-clojure-flavor ()
+  "Determine the flavor of the project."
+  (pcase inf-clojure-repl-flavor
+    (lumo (inf-clojure--lumo-repl-command))
+    (_ nil)))
```

Comment:
The point is that you can't really detect this, you can only set it via some dir-local for instance. That's why I'd rather simply leave people to configure how to start the REPL, run some eval after the REPL starts to check what's its type and use the result to set its flavour. 

Alternatively there could be some commands like `inf-clojure-start-lumo` and so on, but I generally prefer to avoid having an excessive amount of commands.

So, I assume people would simply change `inf-clojure-lein-cmd` or whatever to a command starting a lumo repl if they want to do this.